assets/js/enrollmentForm.js

```diff
@@ -190,26 +190,15 @@ ok3.onclick = function () {
 	
 	hideAllWarnings();
 	emailAddress = inputEmail.value.trim(); // collect email
-	name = inputName.value.trim(); // collect name
-
-	if (name.length === 0) {
-		inputName.style.border = "1px solid red";
-	}
-	else {
-		inputName.style.border = "1px solid #00A22C";
-	}
 	
 	if (emailAddress.length === 0) {
 		inputEmail.style.border = "1px solid red";
+		return;
 	}
 	else {
 		inputEmail.style.border = "1px solid #00A22C";
 	}
```

Comment:
Remove the else block and just have `inputEmail.style.border = "1px solid #00A22C";` since we're returning from the `if` statement